src/pip/_internal/cli/cmdoptions.py

```diff
@@ -749,6 +749,16 @@ def _handle_no_cache_dir(
     "if this option is used.",
 )
 
+check_build_deps: Callable[..., Option] = partial(
+    Option,
+    "--check-build-deps",
+    "--check-build-dependencies",
```

Comment:
Let's only keep the long-form here. There's no reason to add multiple flags for a single piece of functionality. :)

```suggestion
    "--check-build-dependencies",
```